File: Python/examples/put_bucket.py
# -*- coding=utf-8

from qcloud_cos import CosConfig
from qcloud_cos import CosS3Client
from qcloud_cos import CosServiceError
from qcloud_cos import CosClientError

secret_id = 'COS_SECRETID'     # 替换为用户的secret_id
secret_key = 'COS_SECRETKEY'     # 替换为用户的secret_key
region = 'COS_REGION'    # 替换为用户的region
token = None               # 使用临时密钥需要传入Token，默认为空,可不填
config = CosConfig(Region=region, SecretId=secret_id, SecretKey=secret_key, Token=token)  # 获取配置对象
client = CosS3Client(config)

# 创建存储桶
def put_bucket():
    #.cssg-snippet-body-start:[put-bucket]
    response = client.create_bucket(
        Bucket='examplebucket-1250000000'
    )
    
    #.cssg-snippet-body-end

# 创建存储桶
def put_bucket_comp():
    #.cssg-snippet-body-start:[put-bucket-comp]
    response = client.create_bucket(
        Bucket='examplebucket-1250000000'
    )
    
    #.cssg-snippet-body-end

#.cssg-methods-pragma


# 创建存储桶
put_bucket()

# 创建存储桶
put_bucket_comp()

#.cssg-methods-pragma

File: php/examples/PutBucket.php
<?php

use Qcloud\Cos\Client;
use Qcloud\Cos\Exception\ServiceResponseException;

class PutBucket
{
    private $cosClient;

    private $uploadId;
    private $eTag;
    private $versionId;

    // 创建存储桶
    protected function putBucket() {
        $cosClient = $this->cosClient;
        //.cssg-snippet-body-start:[put-bucket]
        try {
            $bucket = "examplebucket-1250000000"; //存储桶名称 格式：BucketName-APPID
            $result = $cosClient->createBucket(array('Bucket' => $bucket));
            //请求成功
            print_r($result);
        } catch (\Exception $e) {
            //请求失败
            echo($e);
        }
        
        //.cssg-snippet-body-end
    }

	//.cssg-methods-pragma

    protected function init() {
        $secretId = "COS_SECRETID"; //"云 API 密钥 SecretId";
        $secretKey = "COS_SECRETKEY"; //"云 API 密钥 SecretKey";
        $region = "COS_REGION"; //设置一个默认的存储桶地域
        $this->cosClient = new Qcloud\Cos\Client(
            array(
                'region' => $region,
                'schema' => 'https', //协议头部，默认为http
                'credentials'=> array(
                    'secretId'  => $secretId ,
                    'secretKey' => $secretKey)));
    }

    public function mPutBucket() {
        $this->init();

        // 创建存储桶
        $this->putBucket();

	    //.cssg-methods-pragma
    }
}
?>